addons/account_edi_ubl/data/ubl_templates.xml

```diff
@@ -74,7 +74,7 @@
                 xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2"
                 xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2">
                 <cbc:ID t-esc="line._origin.id"/>
-                <cbc:Note t-if="line.discount" t-esc="_('Discount (%s %)') % line.discount"/>
+                <cbc:Note t-if="line.discount">Discount (<t t-esc='line.discount'/> %)</cbc:Note>
```

Comment:
useless diff?

Reply:
I removed the translation because it's not possible to call it from Qweb anymore. I checked at other places in Odoo and the fact that this is not translated is not a problem.